Components/ProductListing-filters/productlist.css

```diff
@@ -0,0 +1,191 @@
+body {
+    background-color: var(--example-bg-color);
+}
+
+.logo-input {
+    display: flex;
+    width: 70%;
+}
+
+.search-input-container {
+    display: flex;
+    align-items: center;
+    width: 60%;
+    padding: 0 1rem;
+}
+
+.input-icon {
+    position: relative;
+    right: 3rem;
+    padding: 5px;
+}
+
+.header-input {
+    padding: 13px;
+    width: 90%;
+    border-radius: 3rem;
+    border: none;
+}
+
+.basic {
+    border: 0px;
+}
+
+.number-one {
+    background-color: var(--alert-primary);
+    height: 1.3rem;
+    width: 1.3rem;
+    border-radius: 50%;
+    position: relative;
+    left: -0.9rem;
+    margin-right: -1.3rem;
+    bottom: 0.8rem;
+    border: 3px solid var(--white);
+    font-size: 12px;
+    font-weight: bolder;
+    color: var(--white);
+    text-align: center;
+}
+
+.link-button {
+    text-decoration: none;
+    background-color: var(--white);
+    color: var(--black);
+    padding: 1rem;
+    margin: 1rem;
+}
+
+.nav-icon-link {
+    text-decoration: none;
+    padding: 0;
+}
+
+.btn-link .link-button {
+    color: var(--black);
+}
+
+.btn-link {
+    border: none;
+    padding: 0px;
+}
+
+.filter-section-col {
+    display: flex;
+    flex-direction: column;
+    width: 27%;
+    background-color: var(--white);
+}
+
+.price-slider {
+    padding: 10px 2rem;
```

Comment:
use consistent units.. px or rem..this looks a little odd

Reply:
Sure will update the same